scripts/config/pre-commit.yaml

```diff
@@ -3,32 +3,28 @@ repos:
   hooks:
   - id: scan-secrets
     name: Scan secrets
-    entry: ./scripts/githooks/scan-secrets.sh
-    args: ["check=staged-changes"]
+    entry: /bin/env check=whole-history ./scripts/githooks/scan-secrets.sh
```

Comment:
Quick sanity check: is the change from `staged-changes` to `whole-history` here intentional?